src/agent/executor/base.ts

```
@@ -0,0 +1,87 @@
+    throw new Error('init() must be implemented by concrete executor');
+  }
+
+  async createOpenAIClient(configId: string) {
+    // This will be implemented by concrete classes
+    throw new Error('createOpenAIClient() must be implemented by concrete executor');
+  }
+
+  protected async executeBase(
+    context: AgentContext,
+    useJsonSchema = false,
+    responseSchema?: any
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)